Maestro/subscriptions.json

```diff
@@ -668,10 +673,10 @@
           "Arguments": [
             "-CleanupDocker",
             "-UpdateDependenciesParams '",
-              "--user dotnet-maestro-bot",
-              "--email dotnet-maestro-bot@microsoft.com",
-              "--password `$(`$Secrets[`'DotNetMaestroBotGitHubToken`'])",
-              "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/release/2.1/build.xml",
+            "--user dotnet-maestro-bot",
+            "--email dotnet-maestro-bot@microsoft.com",
+            "--password `$(`$Secrets[`'DotNetMaestroBotGitHubToken`'])",
```

Comment:
Please revert the indentation changes here--they indicate that these params are within `-UpdateDependenciesParams '` ... `'`.